cli/src/commands/clean.rs

```diff
@@ -30,35 +30,38 @@ pub struct Clean {
     /// Enables development mode, specify the unique ID of the local node to clean.
     #[clap(long)]
     pub dev: Option<u16>,
-    /// Specify the path to a directory containing the ledger
-    #[clap(long = "path")]
-    pub path: Option<PathBuf>,
+    /// Specify the path to a directory containing the storage database for the ledger.
+    #[clap(long = "storage")]
```

Comment:
as much as it's vague, we might want to not change this, as it could be part of scripts, which would then break

Reply:
fair point, reverted 👍